sections/semantics-embedded-content.include

```diff
@@ -2981,56 +2981,65 @@
 
   In many cases, the image is actually just supplementary, and its presence merely reinforces the
   surrounding text. In these cases, the <code>alt</code> attribute must be
-  present but its value must be the empty string.
+  present but its value must be unset or an empty string.
```

Comment:
"unset or the empty string" is probably more consistent with how we say it elsewhere.